test/services/sift/ApplicationSiftServiceSpec.scala

```diff
@@ -65,11 +65,8 @@ class ApplicationSiftServiceSpec extends ScalaMockUnitSpec {
 
   trait SiftUpdateTest extends TestFixture {
     val progressStatusUpdateBson = BSONDocument(
-      "$set" -> BSONDocument(
-        s"progress-status.${ProgressStatuses.SIFT_COMPLETED}" -> true,
-        s"progress-status-timestamp.${ProgressStatuses.SIFT_COMPLETED}" ->
-          BSONDateTimeHandler.write(DateTimeFactoryMock.nowLocalTimeZone)
-      )
+      s"progress-status.${ProgressStatuses.SIFT_COMPLETED}" -> true,
```

Comment:
![](http://www.reactiongifs.com/wp-content/uploads/2013/10/tim-and-eric-mind-blown.gif)